src/read_pool.rs

```diff
@@ -754,6 +791,12 @@ mod metrics {
             &["name"]
         )
         .unwrap();
+        pub static ref UNIFIED_READ_POOL_RUNNING_THREADS: IntGaugeVec = register_int_gauge_vec!(
+            "tikv_unified_read_pool_thread_count",
+            "The number of running threads in the unified read pool",
```

Comment:
```suggestion
            "The number of running tasks in the unified read pool",
```
 maybe pending task is more readable.

Reply:
Its `threads` in here.